index.html

```diff
@@ -61,7 +61,7 @@
 			// It initiates all other functionality, directly or indirectly
 			var UI = require('./js/rendererjs/uiManager.js');
 			// Need this for devtools to work
-			var Plugins = require('./js/rendererjs/pluginManager.js');
+			var Plugins;
```

Comment:
where does this value eventually get set? I see `plugins` and `ui` being referenced in the other files; don't they need to be lower-cased here as well?